### PR TITLE
docs: Add more answers to questions I got via email

### PR DESCRIPTION
**Pull request by eseidel**
Add more answers to questions I got via email

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -60,6 +60,39 @@ one megabyte to Flutter apps. `flutter build apk --release` vs.
 we can make this smaller when that becomes a priority. If size is a blocker
 for you, please let us know!
 
+### Does code push work with large applications?
+
+Yes. There is no limit on the size of the application that can be patched with
+code push. As noted [What types of changes does Shorebird code push](below),
```
Comment (felangel) [inline docs/faq.md:66]: Can we fix the (below) since I don't think the link will work as is?

File: docs/faq.md [hunk 2]
```diff
@@ -60,6 +60,39 @@ one megabyte to Flutter apps. `flutter build apk --release` vs.
...
+Shorebird can change any Dart code in your application no matter of size.
+
+### What can I use Shorebird code push for?
+
+We've seen a variety of uses, inculding:
```
Comment (felangel) [inline docs/faq.md:71]:
```suggestion
We've seen a variety of uses, including:
```

File: docs/faq.md (hunk 1 above)
Comment (eseidel) [reply docs/faq.md:66]: Oh, i see, I swapped them?

**Review by felangel — APPROVED** [review]
LGTM ✅

**Pull request merged**